doc/21-development.md

```diff
@@ -1389,36 +1385,27 @@ slower but allows for better debugging insights.
 For benchmarks, change `CMAKE_BUILD_TYPE` to `RelWithDebInfo` and
 build inside the `release` directory.
 
-First, off export some generics for Boost.
-
-```bash
-export I2_BOOST="-DBoost_NO_BOOST_CMAKE=TRUE -DBoost_NO_SYSTEM_PATHS=TRUE -DBOOST_LIBRARYDIR=/usr/lib64/boost169 -DBOOST_INCLUDEDIR=/usr/include/boost169 -DBoost_ADDITIONAL_VERSIONS='1.69;1.69.0'"
-```
-
-Second, add the prefix path to it.
+First, override the default prefix path.
 
 ```bash
-export I2_GENERIC="$I2_BOOST -DCMAKE_INSTALL_PREFIX=/usr/local/icinga2"
+export I2_GENERIC="-DCMAKE_INSTALL_PREFIX=/usr/local/icinga2"
 ```
 
-Third, define the two build types with their specific CMake variables.
+Second, define the two build types with their specific CMake variables.
 
 ```bash
 export I2_DEBUG="-DCMAKE_BUILD_TYPE=Debug -DICINGA2_UNITY_BUILD=OFF $I2_GENERIC"
 export I2_RELEASE="-DCMAKE_BUILD_TYPE=RelWithDebInfo -DICINGA2_WITH_TESTS=ON -DICINGA2_UNITY_BUILD=ON $I2_GENERIC"
 ```
 
-Fourth, depending on your likings, you may add a bash alias for building,
+Third, depending on your likings, you may add a bash alias for building,
 or invoke the commands inside:
 
 ```bash
-alias i2_debug="cd /root/icinga2; mkdir -p debug; cd debug; scl enable devtoolset-11 -- cmake $I2_DEBUG ..; make -j2; sudo make -j2 install; cd .."
-alias i2_release="cd /root/icinga2; mkdir -p release; cd release; scl enable devtoolset-11 -- cmake $I2_RELEASE ..; make -j2; sudo make -j2 install; cd .."
+alias i2_debug="cd /root/icinga2; mkdir -p debug; cd debug; cmake $I2_DEBUG ..; make -j2; sudo make -j2 install; cd .."
+alias i2_release="cd /root/icinga2; mkdir -p release; cd release; cmake $I2_RELEASE ..; make -j2; sudo make -j2 install; cd .."
 ```
 
```

Comment:
> Here, I am ending up with two aliases without any further explanation. I just proceeded with calling `i2_debug`.

Well, here you've two different versions (aliases), one for building Icinga in release and one for building in debug mode.

> Otherwise, the following commands would fail as these directories were not created and `icinga2` was not installed.

`icinga2` was not installed? Did you skip this instruction `git clone https://github.com/icinga/icinga2.git && cd icinga2`?

Reply:
> > Otherwise, the following commands would fail as these directories were not created and `icinga2` was not installed.
> 
> `icinga2` was not installed? Did you skip this instruction `git clone https://github.com/icinga/icinga2.git && cd icinga2`?

No, when I am following the documentation, I will end up with two aliases in my shell, both being kinda undocumented and no further instruction. Thus, when proceeding to the next section, everything will fail as I have to execute one of those aliases. However, this is not stated in the manual. (Yes, it wasn't mentioned before either, but now this section was updated and I tried to blindly follow it, which would fail.)

Just put one sentence there, telling the user to execute `i2_debug`.

Reply:
> No, when I am following the documentation, I will end up with two aliases in my shell, both being kinda undocumented and no further instruction.

I mean the sentence the sentence before says the following:

> Third, depending on your likings, you may add a bash alias for building, or invoke the commands inside:

I mean that somewhat implies that you probably want to run one of these aliases, but yes, it's not 100% accurate as adding an alias is not the same as invoking the commands inside. So maybe just do a small rewording of that sentence to make everyone happy?

Reply:
LGTM with #10156 and the recent change, <https://github.com/Icinga/icinga2/compare/7fd1580d31255598daa3a97765299215c83ced42..6da948c83078098818afac6f1d28997a87e312d3>.